docs/validators/staking/staking-smart-contract.md

```diff
@@ -36,9 +36,9 @@ Because this transaction is a call to the Staking smart contract, it passes info
 - `stake` is the name of the smart contract function to be called;
 - `0002` is the number of nodes (unsigned integer, hex-encoded);
 - 45e7131ba....294812f004 is the BLS key of the first node, represented as a 192-character-long hexadecimal string;
-- `67656e65736973` is a reserved placeholder, required after each BLS key;
+- `67656e65736973` is the result of signing the address of the sender of the transaction, using the BLS private key of the validator being added to the transaction;
```

Comment:
Consider replacing "validator" with "first node", to maintain consistency with the line above.

Reply:
Also, please replace `67656e65736973` with placeholders like `<BLS-signature-1>` and `<BLS-signature-2>`, to emphasize that these values are different for each node being added to the transaction (due to different keys).